tc/tc_util.h

```diff
@@ -24,7 +24,7 @@ struct qdisc_util {
 	struct  qdisc_util *next;
 	const char *id;
 	int (*parse_qopt)(struct qdisc_util *qu, int argc,
-			  char **argv, struct nlmsghdr *n, char *dev);
+			  char **argv, struct nlmsghdr *n, const char *dev);
```

Comment:
Yes, let's also add a `const` in the `parse_copt()` signature. The reviewer didn't specify this, but the same logic applies.